### PR TITLE
Update plan-for-teams-live-events.md

### PR DESCRIPTION
**Pull request by jhwang-ms**
Correcting the licenses needed to organize, produce or present a Teams live event.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/teams-live-events/plan-for-teams-live-events.md [hunk 1]
```diff
@@ -35,10 +35,11 @@ Anyone can attend a live event without a license. Read [Admin quick start - Meet
 
 The following prerequisites are required for the user to schedule a Teams live event.
 
-Here are the licenses that must be assigned to produce or present a Teams live event:  
+Here are the licenses that must be assigned to organize, produce or present a Teams live event:
```
Comment (JohanFreelancer9) [inline Teams/teams-live-events/plan-for-teams-live-events.md:38]:
```suggestion
Here are the licenses that must be assigned to organize, produce, or present a Teams live event:  
```

File: Teams/teams-live-events/plan-for-teams-live-events.md [hunk 2]
```diff
@@ -35,10 +35,11 @@ Anyone can attend a live event without a license. Read [Admin quick start - Meet
 
 The following prerequisites are required for the user to schedule a Teams live event.
 
-Here are the licenses that must be assigned to produce or present a Teams live event:  
+Here are the licenses that must be assigned to organize, produce or present a Teams live event:  
 
-- A Microsoft or Office 365 Enterprise E1, E3, or E5 license or an Office 365 Education A3 or A5 license. The exception to this requirement is guest users can present without a license if the other criteria for [guest users](plan-for-teams-live-events.md#guest-to-present) is met.
-- A Microsoft Teams license - this is included in the licenses listed in the first bullet.
+- **To organize:** A Microsoft or Office 365 Enterprise E1, E3, or E5 license, **[or]** a Microsoft or Office 365 Education A3 or A5 license.
```
Comment (JohanFreelancer9) [inline Teams/teams-live-events/plan-for-teams-live-events.md:40]:
```suggestion
- **To organize:** A Microsoft or Office 365 Enterprise E1, E3, or E5 license, **or** a Microsoft or Office 365 Education A3 or A5 license. 
```

File: Teams/teams-live-events/plan-for-teams-live-events.md [hunk 3]
```diff
@@ -35,10 +35,11 @@ Anyone can attend a live event without a license. Read [Admin quick start - Meet
 
 The following prerequisites are required for the user to schedule a Teams live event.
 
-Here are the licenses that must be assigned to produce or present a Teams live event:  
+Here are the licenses that must be assigned to organize, produce or present a Teams live event:  
 
-- A Microsoft or Office 365 Enterprise E1, E3, or E5 license or an Office 365 Education A3 or A5 license. The exception to this requirement is guest users can present without a license if the other criteria for [guest users](plan-for-teams-live-events.md#guest-to-present) is met.
-- A Microsoft Teams license - this is included in the licenses listed in the first bullet.
+- **To organize:** A Microsoft or Office 365 Enterprise E1, E3, or E5 license, **[or]** a Microsoft or Office 365 Education A3 or A5 license. 
+- **To produce or present:** A Microsoft or Office 365 Enterprise E1, E3 or E5 license, **[or]** a Microsoft or Office 365 Education A1, A3 or A5 license. The exception to this requirement is guest users can present without a license if the other criteria for [guest users](plan-for-teams-live-events.md#guest-to-present) is met.
```
Comment (JohanFreelancer9) [inline Teams/teams-live-events/plan-for-teams-live-events.md:41]:
```suggestion
- **To produce or present:** A Microsoft or Office 365 Enterprise E1, E3, or E5 license, **or** a Microsoft or Office 365 Education A1, A3, or A5 license. The exception to this requirement is guest users can present without a license if the other criteria for [guest users](plan-for-teams-live-events.md#guest-to-present) are met.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: Teams/teams-live-events/plan-for-teams-live-events.md (hunk 2 above)
Comment (jhwang-ms) [reply Teams/teams-live-events/plan-for-teams-live-events.md:40]: I was trying to figure out how to underline the "or", and thought the brackets would make that happen.

Comment (JohanFreelancer9) [reply Teams/teams-live-events/plan-for-teams-live-events.md:40]: No, it won't, but I would suggest keeping it bold and changing it to italic for emphasis, as per my edit.

**Comment by scanum** [comment]
@cichur PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**